index.bs

```diff
@@ -1406,19 +1452,20 @@ The [=authenticator data=] structure is a byte array of 37 bytes or more, as fol
     </tr>
 </table>
 
-The RP ID is originally received from the client when the credential is created, and again when an assertion is generated.
-However, it differs from other [=client data=] in some important ways. First, unlike the client data, the RP ID of a credential
-does not change between operations but instead remains the same for the lifetime of that credential. Secondly, it is validated
-by the authenticator during the [=authenticatorGetAssertion=] operation, by verifying that the RP ID associated with the
-requested credential exactly matches the RP ID supplied by the client.
+The [=RP ID=] is originally received from the client when the credential is created, and again when an assertion is generated.
+However, it differs from other [=client data=] in some important ways. First, unlike the client data, the [=RP ID=] of a
+credential does not change between operations but instead remains the same for the lifetime of that credential. Secondly, it is
+validated by the authenticator during the [=authenticatorGetAssertion=] operation, by verifying that the [=RP ID=] associated
+with the requested credential exactly matches the [=RP ID=] supplied by the client, and that the [=RP ID=] [=is a registrable
+domain suffix of or is equal to=] the [=effective domain=] of the RP's [=origin=]'s [=effective domain=].
```

Comment:
"the [=effective domain=] of the RP's [=origin=]'s [=effective domain=]." has two "effective domain"s. Pick one. :) I'd probably use "origin's effective domain" just because it's a bit shorter, but both are correct.